### ISSUE TITLE
visual selector

### ISSUE THREAD

**Issue by louis030195** [mediar-ai/terminator#91]
i wonder if this could be any useful 

like finding a similar chunk of screen on the screen 

probably not the best - only useful when accessibility fails and also might require AI (don't think heuristics are good)
